### ISSUE TITLE
Not enough disk space error translation - Es

### ISSUE THREAD

**Issue by Grantlyk** [FreeCodeCamp/FreeCodeCamp#9650]
Spanish language translation for PR #9647

Added text: "Together with the commands used in Step 11, this adds additional parameters to the
mongod command to allow MongoDB to run in the C9 environment." & "Note: Use the command mongod --port 27017 in place of mongod --port 27017 --dbpath=./data</code> to avoid a duplicate parameter error.--dbpath=./data is already included in the ./mongod command."

**Comment by josevill** [comment]
Added text: "Together with the commands used in Step 11, this adds additional parameters to the
mongod command to allow MongoDB to run in the C9 environment." & "Note: Use the command mongod --port 27017 in place of mongod --port 27017 --dbpath=./data to avoid a duplicate parameter error.--dbpath=./data is already included in the ./mongod command."

Translated text: "Conjuntamente con los comandos utilizados en el paso 11, esto agrega parámetros adicionales al comando mongod para habilitar a MongoDB correr en el entorno C9."
"Nota: Utiliza el comando mongod --port 27017 en lugar de mongod --port 27017 --dbpath=./data para evitar el error de parámetro duplicado. --dbpath=./data ya se encuentra incluido en el comando ./mongod" 

In case you want to keep the error the same replace from "--dbpath=./data ya  se encuentra incluido en el comando ./mongod" to ".--dbpath=./data is already included in the ./mongod command."

**Comment by jhuynh85** [comment]
Are the above translations good to go? If so I can add them to the project. 

Unless you guys want a native Spanish speaker to work on this issue?

**Comment by josevill** [comment]
@jhuynh85 Hi there.
I'm a native Spanish speaker and fluent in English. My translation is accurate 😀

**Comment by jhuynh85** [comment]
@josevill I wasn't calling into question your translations :smiley: I was just wondering if the project admins preferred that a Spanish speaker merge them to the project since my Spanish is minimal

**Comment by jhuynh85** [comment]
Hey guys, I delved into this a little more and I noticed a couple things about Step 13:

- The new text from PR #9647 makes a reference to Step 11, which in the English text corresponds to 

`"Set up your mongod alias by following the directions at <a href='https://community.c9.io/t/setting-up-mongodb/1717' target='_blank'>https://community.c9.io/t/setting-up-mongodb/1717</a>."`

However, this step has been left out in the Spanish text (presumably because the directions in the link are in English), so adding the Spanish translation that we have would make no sense.

- I also noticed the Spanish translation does not match the English. The English says to use `./mongod` while the Spanish says to use `mongod --nojournal`:

**ENGLISH:** `Whenever you run a command that includes mongod on c9.io, use ./mongod instead. Together with the commands used in Step 11, this adds additional parameters to the mongod command to allow MongoDB to run in the C9 environment.`

**SPANISH:** `Siempre que ejecutes un comando que incluya <code>mongod</code> en c9.io, asegúrate de usar la bandera <code>--nojournal</code>. Por ejemplo: <code>mongod --nojournal</code>.`
